src/components/notes/saving-indicator.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useRef, useState } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import { Text } from 'cozy-ui/react/Text'
+
+import styles from 'components/notes/saving-indicator.styl'
+
+// constants
+const sec = 1000
+const min = 60 * sec
+const hour = 60 * min
+const day = 24 * hour
+
+// differents steps for message formating
+// [age, key, unit, coef] :
+// - `age`: the line applies only if the last saved
+// was done less than this `age` ago
+// (only the first match will apply)
+// - `key` is the traduction key for the message
+// - `unit` is the unit in which we will send the time
+// to the translated message
+// - `coef` is  the granularity in which we will send
+// the time (`min, 5` is "by 5 minutes slices")
+const steps = [
+  [10 * sec, 'saved_just_now', sec, 1],
+  [min, 'saved_secs_ago', sec, 10],
+  [2 * min, 'saved_min_ago', min, 1],
+  [5 * min, 'saved_mins_ago', min, 1],
+  [20 * min, 'saved_mins_ago', min, 5],
+  [hour, 'saved_mins_ago', min, 10],
+  [2 * hour, 'saved_hour_ago', hour, 1],
```

Comment:
```suggestion
  [2 * hour, 'saved_hours_ago', hour, 1],
```
?

Reply:
nope.  The condition is "when time < 2 * hours". Which means it will be 1 hour (without "s")